### DISCUSSION TITLE
Fixed versions in compose example

### DISCUSSION THREAD

**Discussion by qwerty287** [woodpecker-ci/woodpecker#2806] [General]
From https://github.com/woodpecker-ci/woodpecker/pull/2752#discussion_r1390197716:

> Do we want to suggest latest here? (Sorry, I know somewhat off-topic but I just saw it here). I think it should be possible to also let this being updated dynamically via renovate without manual work by ourselvse.

and

> you will also never know which version/commit you're actually on. If we suggest such examples, users will use it like this and never change it. Hence I suggest we always put the most recent version into the example.

**Comment by pat-s** [comment]
Adding to this: 

> you will also never know which version/commit you're actually on

Of course, you can look it up in WP itself, and it will tell you, but if the container is recreated or some automation is run, you will get the most recent WP without necessarily wanting it. You/the devs of course know this, but not everyone is an expert/cares about this and my point is just: we shouldn't use `latest` in our own docs for educational purposes :)

**Reply by zc-devs** [reply]
> we shouldn't use latest in our own docs for educational purposes

Does it apply to the `main` (Next) version?

**Comment by qwerty287** [comment]
We now have an internal update check which will notify you about updates. I don't think it's necessary to use fixed versions - if your instance in outdated you'll get the notification anyways.

**Reply by pat-s** [reply]
The check helps, but I still believe we should not use `:latest` in our examples. Even if currently `latest` resolves to the latest semver tagged version. But no need for further discussion, I am aware that different opinions exist and if `latest` would resolve to `next`, this would be much more/a real problem, but in the current case it is more cosmetical.

**Reply by qwerty287** [reply]
@pat-s So we can close this?
